plugins/gateway/kong/README.md

```diff
@@ -56,7 +56,7 @@ If you just want to try it out with a demo application, and you don't have kong
        KONG_GATEWAY_DEPLOYMENT_NAMESPACE=kong \
        KONG_GATEWAY_INGRESS_NAME=catalogue \
        KONG_GATEWAY_INGRESS_NAMESPACE=sock-shop \
-       UPSTREAM_TELEMETRY_HOST_NAME=apiclarity-apiclarity.apiclarity:9000 \
+       UPSTREAM_TELEMETRY_ADDRESS=apiclarity-apiclarity.apiclarity:9000 \
```

Comment:
Is documentation needed for `TRACE_SAMPLING_ENABLED` and  `TRACE_SAMPLING_ADDRESS` ?

Reply:
Probably yes. Not part of this task though